trpc/router/goal.ts

```diff
@@ -453,6 +453,10 @@ export const goal = router({
         }
     }),
     update: protectedProcedure.input(goalUpdateSchema).mutation(async ({ ctx, input }) => {
+        if (input.estimate && input.estimate.date) {
+            input.estimate.date = convertDateByLocale(input.estimate.date, 'en');
```

Comment:
why not use ISO? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

Reply:
input.estimate.date in ru locale isn't date time string format and I abandoned this idea too quickly